fawkes/cli/cli.py

```diff
@@ -79,6 +81,8 @@ def define_arguments(parser):
         send_email.send_email(app_config_file)
     elif action == FawkesActions.PUSH_ELASTICSEARCH:
         elasticsearch.push_data_to_elasticsearch(app_config_file)
+    elif action == FawkesActions.QUERY_ELASTICSEARCH:
+        elasticsearch.query_from_elasticsearch(app_config_file)
```

Comment:
Not providing any params, will result in fetching all the data.

Can you pls add an additional argument (optional argument) to give the query itself from the command line.

You can name it `--query` and  `-q` (shorthand)

Reply:
@MadaraUchiha-314 Added the query as well as format argument for this